contracts/FT-voting-simple/ProposalTarget.sol

```diff
@@ -1,13 +1,20 @@
 // SPDX-License-Identifier: GPL-3.0
 pragma solidity ^0.8.9;
 
-import './RIFGovernorFT.sol';
-
-contract ProposalTarget {
-    RIFGovernorFT governor;
+import '@openzeppelin/contracts/access/Ownable.sol';
+import '@openzeppelin/contracts/governance/IGovernor.sol';
 
+interface IProposalTarget {
     event ProposalProcessed(uint256 proposalId);
 
+    function setGovernor(IGovernor _governor) external;
+
+    function onProposalExecution(uint256 _proposalId) external;
+}
+
+contract ProposalTarget is IProposalTarget, Ownable {
+    IGovernor public governor;
+
     modifier governorOnly() {
```

Comment:
I suggest that this modifier be renamed to `onlyGovernor` to be consistent with OZ's `onlyOwner` naming pattern.